
base_station: 41AF1B21
boat1: 417E6267
boat2: 41AF1BEE 
boat3: 417E6274
boat4: 41AF1A01